language-support/java/codegen/src/it/daml/Lib.daml

```diff
@@ -13,6 +13,7 @@ import Tests.VariantTest()
 
 import Tests.Import()
 import Tests.Escape()
-import Tests.MapTest()
 import Tests.Serializable()
+-- the two next line has to be commented out to regenerate LF <= 1.3
```

Comment:
If we recompile the sources in the DAR, which we should always do anyway, this comment isn't really necessary anymore. I would just leave the file unchanged.